package.json

```diff
@@ -7,19 +7,46 @@
     "author": "CXL <leho@cxl.com>",
     "license": "MIT",
     "peerDependencies": {
-        "eslint": "^7.7.0",
-        "husky": "^4.2.5",
-        "lint-staged": "^10.2.13",
-        "prettier": "^2.1.1"
+        "@10up/eslint-config": "^2.1.1",
+        "@10up/stylelint-config": "^1.0.11",
+        "@wordpress/eslint-plugin": "^8.0.2",
+        "babel-eslint": "^10.1.0",
+        "eslint": "^7.17.0",
+        "eslint-config-airbnb": "^18.2.1",
+        "eslint-config-airbnb-base": "^14.2.1",
+        "eslint-config-prettier": "^7.1.0",
+        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-jsdoc": "^31.6.0",
+        "eslint-plugin-jsx-a11y": "^6.4.1",
+        "eslint-plugin-prettier": "^3.3.1",
+        "eslint-plugin-react": "^7.22.0",
+        "eslint-plugin-react-hooks": "^4.2.0",
+        "husky": "^4.3.7",
+        "lint-staged": "^10.5.3",
+        "prettier": "npm:wp-prettier@2.0.5",
+        "stylelint": "^13.8.0"
     },
     "scripts": {
         "postinstall": "node postinstall.js"
     },
     "devDependencies": {
-        "eslint": "^7.7.0",
-        "eslint-config-prettier": "^6.11.0",
-        "husky": "^4.2.5",
-        "lint-staged": "^10.2.13",
-        "prettier": "^2.1.1"
+        "@10up/eslint-config": "^2.1.1",
+        "@10up/stylelint-config": "^1.0.11",
+        "@wordpress/eslint-plugin": "^8.0.2",
+        "babel-eslint": "^10.1.0",
+        "eslint": "^7.17.0",
+        "eslint-config-airbnb": "^18.2.1",
+        "eslint-config-airbnb-base": "^14.2.1",
+        "eslint-config-prettier": "^7.1.0",
+        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-jsdoc": "^31.6.0",
+        "eslint-plugin-jsx-a11y": "^6.4.1",
+        "eslint-plugin-prettier": "^3.3.1",
+        "eslint-plugin-react": "^7.22.0",
+        "eslint-plugin-react-hooks": "^4.2.0",
```

Comment:
I don't think most if any of these are necessary for dev-ing this package itself.

Reply:
> I don't think most if any of these are necessary for dev-ing this package itself.

That's my view as well, but they were auto installed as deps, so left them as is, in case update of package might try to install them again, so ignored them for now.

Reply:
"Auto-installed", how exactly?

Reply:
Either way, should remove.